src/page.js

```diff
@@ -61,13 +61,14 @@ export class Page {
   // Returns nothing.
   dispatch(data = {}) {
     const raw = this.recognize();
+    if (raw === void(0) || raw === null) {
+      throw new Error('<body> does not have a [data-page] attribute');
```

Comment:
Master should be green by now - I pushed some fixes earlier. You can rebase your branch and everything should be fine.